modules/data/redis-cache/main.bicep

```diff
@@ -0,0 +1,250 @@
+@description('Required. The name of the Redis cache resource.')
+param name string
+
+@description('Optional. The location to deploy the Redis cache service.')
+param location string = resourceGroup().location
```

Comment:
```suggestion
@description('Required. The location to deploy the Redis cache service.')
param location string
```

Reply:
we make the location required to prevent upstream static analysis issues.

Reply:
Good Catch. Done